opencti-platform/opencti-graphql/src/domain/connector.js

```diff
@@ -130,6 +132,41 @@ export const connectorDelete = async (context, user, connectorId) => {
   await notify(BUS_TOPICS[ABSTRACT_INTERNAL_OBJECT].DELETE_TOPIC, element, user);
   return element.internal_id;
 };
+
+export const connectorTriggerUpdate = async (context, user, connectorId, input) => {
+  const conn = await storeLoadById(context, user, connectorId, ENTITY_TYPE_CONNECTOR);
+  if (!conn) {
+    throw FunctionalError('Cant find element to update', { id: connectorId, type: ENTITY_TYPE_CONNECTOR });
+  }
+  if (!['INTERNAL_ENRICHMENT', 'INTERNAL_IMPORT_FILE'].includes(conn.connector_type)) {
+    throw FunctionalError('Update is only possible on internal enrichment or import file connectors types');
+  }
+  const supportedInputKeys = ['connector_trigger_filters', 'auto'];
```

Comment:
Why is 'auto' a supported input key, since we currently only change the filters on the frontendside? Is it going to be added in the future?

Reply:
yes I added this in prevision of being able to change the auto param directly in the front, but maybe I could remove it

Reply:
I think it's OK to keep, but I was mostly wondering how it would work with the auto coming from the connector config file? Won't the auto configured with this method be overriden everytime the connector is restarted and registers to the API with its config from config file?

Reply:
done